### DISCUSSION TITLE
Inbox & Explore question

### DISCUSSION THREAD

**Discussion by bearded-papa** [usememos/memos#2721] [Q&A]
What is the Inbox & Explor option for?

I have looked through the documentation and have not found any reference to them and on my instance there is no content in these sections

**Comment by n1njaznutz** [comment]
Not entirely sure, due to the lack of documentation relating to these concepts, however, I think the Inbox and Explore are utilised when you have other users on your instance/server.

**Reply by ahit976** [reply]
If you have other accounts in Memos, you will find the memos that have been set to workspace or public in the Explore. 
But still not knowing the Inbox's function.

**Reply by IanHarker** [reply]
Yes please - what is the inbox for ? As mentioned no documentation on this area

**Reply by RoccoSmit** [reply]
Please see https://github.com/orgs/usememos/discussions/2892#discussioncomment-10517139
